KnockKnock-iOS/Repository/ChellengeRepository.swift

```diff
@@ -10,17 +10,19 @@ import Foundation
 import Alamofire
 
 protocol ChallengeRepositoryProtocol {
-  func fetchChellenge() -> [Challenge]
+  func fetchChellenge(completionHandler: @escaping ([Challenges]) -> Void)
 }
 
 final class ChallengeRepository: ChallengeRepositoryProtocol {
-  
-  /// 해다 영역에서 네트워크 통신하여 데이터를 Fetch 합니다.
-  func fetchChellenge() -> [Challenge] {
-    return [
-      Challenge(title: "Gogo 챌린지", contents: "쓰레기를 줄이쟈쓰레기를 줄이쟈쓰레기를 줄이쟈쓰레기를 줄이쟈"),
-      Challenge(title: "제로웨이스트", contents: "쓰레기를 줄이쟈쓰레기를 줄이쟈쓰레기를 줄이쟈쓰레기를 줄이쟈"),
-      Challenge(title: "오호호호호호호", contents: "쓰레기를 줄이쟈쓰레기를 줄이쟈쓰레기를 줄이쟈쓰레기를 줄이쟈")
-    ]
+  func fetchChellenge(completionHandler: @escaping ([Challenges]) -> Void) {
+    KKNetworkManager.shared
+      .request(object: [Challenges].self,
+               router: KKRouter.getChallengeResponse,
+               success: { response in
+        completionHandler(response)
+      },
+               failure: { response in
+        print(response)
+      })
```

Comment:
```suggestion
    KKNetworkManager.shared.request(
      object: [Challenges].self,
      router: KKRouter.getChallengeResponse,
      success: { response in
        completionHandler(response)
      }, failure: { response in
        print(response)
      })
```
이렇게 가이드상 맞아 보입니다! 
이건 조금 애매할 수도 있는데 다음과 같이 하시죠!